src/Claims.ts

```diff
@@ -227,32 +225,43 @@ export class Claims {
    *   If the scope is an asset DID, the corresponding ticker is returned as well
    *
    * @param opts.target - identity for which to fetch claim scopes (optional, defaults to the current Identity)
-   *
-   * @note a null scope means the Identity has scopeless claims (like CDD for example)
-   * @note uses the middleware
    */
   public async getClaimScopes(opts: { target?: string | Identity } = {}): Promise<ClaimScope[]> {
     const { context } = this;
     const { target } = opts;
 
     const did = await getDid(target, context);
 
-    const {
-      data: { scopesByIdentity: scopes },
-    } = await context.queryMiddleware<Ensured<Query, 'scopesByIdentity'>>(
-      scopesByIdentity({ did })
-    );
+    const identityClaimsFromChain = await context.getIdentityClaimsFromChain({
+      targets: [did],
+      claimTypes: [
+        ClaimType.Accredited,
+        ClaimType.Affiliate,
+        ClaimType.Blocked,
+        ClaimType.BuyLockup,
+        ClaimType.Exempted,
+        ClaimType.InvestorUniqueness,
+        ClaimType.Jurisdiction,
+        ClaimType.KnowYourCustomer,
+        ClaimType.SellLockup,
+      ],
```

Comment:
To avoid having to manually write all claim types, I propose you make `claimTypes` optional in `getIdentityClaimsFromChain` and have it default to `Object.values(ClaimType)` (add a comment on that line saying "no value means **ALL** claim types")

Reply:
NoData and CustomerDueDiligence are not present